app/jobs/dedupe_job.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+# Removes duplicate stragglers
+class DedupeJob < ApplicationJob
+  TMP_DUPE_BACKUPS_PATH = Rails.root.join("tmp/odk_dupes_backup")
+
+  def perform
+    backup_duplicate_xml
+    destroy_duplicates!
+    clean_up
+  end
+
+  private
+
+  def dupe_codes
+    return false if Response.dirty_dupe.empty?
```

Comment:
it seems unusual to return `false` since this method normally returns an array.

actually maybe we can remove this return statement entirely? because `tuples` will be an empty array if there are no dirty dupes, so then `group_by` etc. will just return the empty array below.